css/index.css

```diff
@@ -1 +1,234 @@
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  font: inherit;
+  vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
+}
+body {
+  line-height: 1;
+}
+ol,
+ul {
+  list-style: none;
+}
+blockquote,
+q {
+  quotes: none;
+}
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
+  content: '';
+  content: none;
+}
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+/* Set every element's box-sizing to border-box */
+* {
+  box-sizing: border-box;
+}
+html,
+body {
+  height: 100%;
+}
+/******** my code ************/
+.wrapper {
+  margin: 2% 10% 2% 10%;
+}
+.wrapper .top-section {
+  display: flex;
+  flex-flow: column wrap;
+  align-content: center;
+  margin-bottom: 8%;
+  border: 1px solid lightslategray;
+  padding-top: 3%;
+  padding-bottom: 2.3%;
+}
+.wrapper .top-section nav {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: space-between;
+  margin: 0 2.5% 0 2.5%;
+  color: darkblue;
+  margin-bottom: 10%;
+}
+@media (max-width: 500px) {
+  .wrapper .top-section nav {
+    display: flex;
+    flex-flow: column wrap;
+    width: 100%;
+    align-items: center;
+  }
+}
+.wrapper .top-section nav a {
+  border: 1px solid lightslategray;
+}
+@media (max-width: 500px) {
+  .wrapper .top-section nav a {
+    width: 95%;
+    border: 1px solid lightslategray;
+    box-sizing: border-box;
+    padding: 3% 1% 3% 40%;
+  }
+}
+.wrapper .top-section nav a:hover {
+  background-color: 1px solid lightslategray;
+  color: aqua;
+}
+.wrapper .top-section .text-control {
+  display: flex;
+  flex-flow: column nowrap;
+  margin-right: 10%;
+  margin-left: 10%;
+}
+.wrapper .top-section .text-control p {
+  margin-bottom: 5%;
+  width: 99%;
+}
+.wrapper .top-section .text-control .tc-btn {
+  display: flex;
+  flex-flow: row nowrap;
+  justify-content: center;
+}
+.wrapper .top-section .text-control .tc-btn button {
+  width: 20%;
+}
+.wrapper .bottom-section {
+  display: flex;
+  flex-flow: row wrap;
+}
+.wrapper .bottom-section .wide {
+  display: flex;
+  flex-flow: column wrap;
+  width: 100%;
+  border: 1px solid black;
+  padding: 3%;
+}
+.wrapper .bottom-section .columns {
+  display: flex;
+  flex-flow: row wrap;
+  width: 100%;
+  justify-content: space-around;
+  margin-top: 7%;
+}
+@media (max-width: 500px) {
+  .wrapper .bottom-section .columns {
+    display: column wrap;
+  }
+}
+.wrapper .bottom-section .a-column {
+  width: 47%;
+  border: 1px solid lightslategray;
+  padding: 2%;
+}
+@media (max-width: 500px) {
+  .wrapper .bottom-section .a-column {
+    width: 98%;
+    margin-bottom: 1.5%;
+  }
+}
```

Comment:
Task 3: Project Objectives
 - Variables: Incorporate variables in your project for color and font stacks.  - done!
 - Nesting: Every selector should be nested inside the main container. Avoid having global styles unless they are element level. - done!
 - Mixins: Create 2 mixins of your choosing. Hint: It's super helpful to use flexbox properties in mixins - done!
 Mobile: Use nested-at rules to provide a mobile version of your resume. Use 500px as a max-width for mobile. - done!